src/libraries/System.Private.CoreLib/src/System/Text/Utf8Span.Searching.cs

```diff
@@ -239,7 +239,7 @@ private unsafe bool TryFind(Utf8Span value, StringComparison comparisonType, out
 
                     case StringComparison.OrdinalIgnoreCase:
                         // TODO_UTF8STRING: Can probably optimize this case.
-                        compareInfo = CompareInfo.Invariant;
+                        compareInfo = CultureInfo.InvariantCulture.CompareInfo;
```

Comment:
Bouncing through `CultureInfo.InvariantCulture.CompareInfo` instead of `CompareInfo.Invariant` will suppress JIT / AOT optimizations that would otherwise take place. In the netcoreapp5.0 code paths we should continue to use `CompareInfo.Invariant` directly.

(Same comment elsewhere in this method.)